### PR TITLE
 QUEST_JOBSTEP.tsv [0-200]

### PR DESCRIPTION
**Pull request by Azhurel**
Minor tweaks for sentence flow and consistency.

Miner's Village -> Miners' Village, as per the glossary.

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
...
 QUEST_JOBSTEP_20150317_000135	As compensation for seeking cooperation, just support 5 boards of wooden buckler type.{nl}Defeating Upents at Indigo Trees Plateau will be enough.
 QUEST_JOBSTEP_20150317_000136	Thank you for the supplies, but the Highlander Master wouldn't have sent you for nothing. {nl}Tell me what you want. {nl}
 QUEST_JOBSTEP_20150317_000137	An overall cooperation between the Masters. Alright. {nl}Tell him that the Peltastas are for it. 
 QUEST_JOBSTEP_20150317_000138	If the Masters unite, it won't be long until the kingdom rises again. {nl}You did a good job. I will teach you my swordsmanship.
 QUEST_JOBSTEP_20150317_000139	That is enough. {nl}Get the Peltasta's cooperation. 
-QUEST_JOBSTEP_20150317_000140	We Peltastas promised the Highlanders we would lend our force to rebuild the kingdom.{nl}So if you also want to learn from me, you should be also responsible.{nl}
+QUEST_JOBSTEP_20150317_000140 We Peltastas promised the Highlanders we would lend our forces in rebuilding the kingdom.{nl}So if you want to learn from me, you should also be responsible.{nl}
```
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: It looks like you messed up a tab here.

I believe "force to rebuild" is correct here, although you could also use "forces to rebuild".  "Forces in rebuilding" is a bit uncommon, and might be incorrect.

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -152,29 +152,29 @@ QUEST_JOBSTEP_20150317_000151	I need to gather the energy of fire for my researc
...
 QUEST_JOBSTEP_20150317_000170	Then fight against Mentiwoods at the Seven Hues Mesa.{nl}You may learn something from doing this.
 QUEST_JOBSTEP_20150317_000171	The most important thing for a Priest is to believe in the Goddess, and prove it. {nl}Exorcise the soldiers possessed with evil spirits in the name of Goddess. 
 QUEST_JOBSTEP_20150317_000172	That soldier is being taken care of in the Miners' Village. {nl}Your sincerity will become your strength. 
 QUEST_JOBSTEP_20150317_000173	Your sincerity has been proven. {nl}We will walk together in the name of the Goddess. 
 QUEST_JOBSTEP_20150317_000174	Learning magic from me comes after helping my research. {nl}Let's talk about the talent problem afterwards. 
-QUEST_JOBSTEP_20150317_000175	First get me Skins of Grolls from the Spine Heart Dale, Feelers of Chafperors from Fallen Worry Forest, {nl}And lastly the nuclueses from Jukopus. I am counting on you.
+QUEST_JOBSTEP_20150317_000175	First get me Skins of Grolls from the Spine Heart Dale, Feelers of Chafperors from Fallen Worry Forest, {nl}And lastly, the nuclueses from Jukopus. I am counting on you.
```
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: Hmm, looking at this line again, "nucleuses" should become "nuclei"

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -185,15 +185,15 @@ QUEST_JOBSTEP_20150317_000184	If it's for the Cleric Master, take as much holy w
 QUEST_JOBSTEP_20150317_000185	Please send my regards to the Cleric Master. {nl}May the blessings of the Goddess be with you. 
 QUEST_JOBSTEP_20150317_000186	Please send them my regards while you get holy water from the masters. 
 QUEST_JOBSTEP_20150317_000187	Service is what you do for others, but it is also a way of cleansing your own reflection. {nl}I hope your wandering soul will be cleansed as you treat the patients. 
-QUEST_JOBSTEP_20150317_000188	Good job. {nl}I will teach you the faith of Clerics as promised. 
-QUEST_JOBSTEP_20150317_000189	I think that you may not have a great spear right away {nl}but you must have a spear that you are strongly attached to. {nl}What do you think about it? 
-QUEST_JOBSTEP_20150317_000190	Spears are the pride of Hoplites. {nl}A strong spear is something that Hoplites can boast like their name.
+QUEST_JOBSTEP_20150317_000188 Good job. {nl}I will teach you the faith of Clerics, as promised.
```
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: Looks like a tab was removed here.

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
...
 QUEST_JOBSTEP_20150317_000135	As compensation for seeking cooperation, just support 5 boards of wooden buckler type.{nl}Defeating Upents at Indigo Trees Plateau will be enough.
 QUEST_JOBSTEP_20150317_000136	Thank you for the supplies, but the Highlander Master wouldn't have sent you for nothing. {nl}Tell me what you want. {nl}
 QUEST_JOBSTEP_20150317_000137	An overall cooperation between the Masters. Alright. {nl}Tell him that the Peltastas are for it. 
 QUEST_JOBSTEP_20150317_000138	If the Masters unite, it won't be long until the kingdom rises again. {nl}You did a good job. I will teach you my swordsmanship.
 QUEST_JOBSTEP_20150317_000139	That is enough. {nl}Get the Peltasta's cooperation. 
-QUEST_JOBSTEP_20150317_000140	We Peltastas promised the Highlanders we would lend our force to rebuild the kingdom.{nl}So if you also want to learn from me, you should be also responsible.{nl}
+QUEST_JOBSTEP_20150317_000140  We Peltastas promised the Highlanders we would lend our forces to rebuild the kingdom.{nl}So if you want to learn from me, you should also be responsible.{nl}
```
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: Needs to be a tab, instead of two spaces.

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -185,15 +185,15 @@ QUEST_JOBSTEP_20150317_000184	If it's for the Cleric Master, take as much holy w
 QUEST_JOBSTEP_20150317_000185	Please send my regards to the Cleric Master. {nl}May the blessings of the Goddess be with you. 
 QUEST_JOBSTEP_20150317_000186	Please send them my regards while you get holy water from the masters. 
 QUEST_JOBSTEP_20150317_000187	Service is what you do for others, but it is also a way of cleansing your own reflection. {nl}I hope your wandering soul will be cleansed as you treat the patients. 
-QUEST_JOBSTEP_20150317_000188	Good job. {nl}I will teach you the faith of Clerics as promised. 
-QUEST_JOBSTEP_20150317_000189	I think that you may not have a great spear right away {nl}but you must have a spear that you are strongly attached to. {nl}What do you think about it? 
-QUEST_JOBSTEP_20150317_000190	Spears are the pride of Hoplites. {nl}A strong spear is something that Hoplites can boast like their name.
+QUEST_JOBSTEP_20150317_000188  Good job. {nl}I will teach you the faith of Clerics, as promised.
```
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: Needs to be a tab, instead of two spaces.

**Comment by Azhurel** [comment]
I swear I'm putting tabs in there. It's driving me nuts.

**Comment by grillo-delmal** [comment]
I think your problem is your editor "indentation" setting 
In the case of the GitHub editor:
- The combo-box that says Spaces, change it to Tabs
- The combo-box right next to it, there is a number, that marks how many characters will the Tab cover

This is a function that is useful for programmers when writing neat code (you can research on the religious war between tabs and space indentation)

If you still have problems, i sent you a pull request to your repo with the problem fixed, you just need to accept it and it will merge the changes

**Comment by Azhurel** [comment]
I ~think~ I got it, but check just in case.

**Comment by grillo-delmal** [comment]
Still spaces, try this

https://github.com/Azhurel/EnglishTranslation/pull/2

On that link I fixed the problem for you, you can merge it by merging the pull request (press the Merge pull request" button)

you can get more help on this function here: https://help.github.com/articles/merging-a-pull-request/

**Comment by Azhurel** [comment]
Aha! Okay, thanks. You just need to sign the CLA. :+1:

**Comment by grillo-delmal** [comment]
signed :+1:

**Comment by grillo-delmal** [comment]
You still need to add the $ signs to mark your edits

File: QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -2,22 +2,22 @@ QUEST_JOBSTEP_20150317_000001	Bokor. You’ve chosen a difficult path. {nl}Refer
...
+QUEST_JOBSTEP_20150317_000010	$Hurry. {nl} It's a simple challenge, but Vubbes are a threat to the Miners' Village.
 QUEST_JOBSTEP_20150317_000011	Good work, you've completed your initiation in becoming a Highlander. {nl} Keep up the hard work and dedication.
 QUEST_JOBSTEP_20150317_000012	Pyromancer Master
 QUEST_JOBSTEP_20150317_000013	I can see that you want to learn magic from me, but I can't teach just anyone. {nl}I will see how well you can assist my research and then decide. 
 QUEST_JOBSTEP_20150317_000014	Good work. Thanks to you I can start the next step of my research. {nl}I will teach you Pyromancer magic as promised. 
 QUEST_JOBSTEP_20150317_000015	Good, you're here. I need you to defeat the evil spirits that I will summon. {nl}I will record the unique wavelength that is transmitted in the process. 
-QUEST_JOBSTEP_20150317_000016	I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master, who deals with the spirits. 
+QUEST_JOBSTEP_20150317_000016	$I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master, who deals with spirits.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:16]: Isn't this missing an 'a'? "I am doing **a** research on the 4th state of matter."

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
 QUEST_JOBSTEP_20150317_000061	Huh, this is the tool I lost. {nl}Thank you. Where has it been?
 QUEST_JOBSTEP_20150317_000062	Did you forget about the donation? {nl}It's 5000 silver.
 QUEST_JOBSTEP_20150317_000063	Cleric Master
-QUEST_JOBSTEP_20150317_000064	Thank you in the name of Goddess.{nl}I promise the donations will be used for those injured by monsters. {nl}
-QUEST_JOBSTEP_20150317_000065	I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now {nl}
+QUEST_JOBSTEP_20150317_000064	$Thank you, in the name of the Goddess.{nl}I promise the donations will be used for those injured by monsters. {nl}
+QUEST_JOBSTEP_20150317_000065	$I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now. {nl}
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv]: Missing a space after the ellipse. 
I'd also write this as "But there is still a small load left on my mind... and I will get rid of it now."

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
...
 QUEST_JOBSTEP_20150317_000072	This is of good quality. The Hunter Master will be pleased. {nl}You can take the book. {nl}
-QUEST_JOBSTEP_20150317_000073	Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you so better hurry.
-QUEST_JOBSTEP_20150317_000074	The Archer Master in Klaipeda has the next book. {nl}He's not a mean person so he will give you the book if you ask nicely. {nl}
-QUEST_JOBSTEP_20150317_000075	He is helping with the repair of the damaged city wall. {nl}I heard there are rocks delivered to the Miner's Village, so bringing him that rock can be a way to ask. 
+QUEST_JOBSTEP_20150317_000073	$Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you, so better hurry.
+QUEST_JOBSTEP_20150317_000074	$The Archer Master in Klaipeda has the next book. {nl}He's not a mean person, so he will give you the book if you ask nicely. {nl}
+QUEST_JOBSTEP_20150317_000075	$He is helping with the repair of the damaged city wall. {nl}I heard rocks were delivered to the Miners' Village, and bringing him a rock can be a way to ask.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:75]: I think "boulder" would fit better here, when I hear "rock" I picture a small rock.

File: QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
...
 QUEST_JOBSTEP_20150317_000078	Don't ask why. Work is work, and it's a fact that we don't want to see each other. 
 QUEST_JOBSTEP_20150317_000079	This pouch...Did you come from the Hunter Master? {nl}Here. Give her this book. 
 QUEST_JOBSTEP_20150317_000080	I don't want to act mean when she's acting humble. 
 QUEST_JOBSTEP_20150317_000081	This leather will be enough. {nl}Go to the Quarrel Shooter Master before someone else takes it. 
 QUEST_JOBSTEP_20150317_000082	This rock will be useful in repairing the wall. {nl}Bring it to the Archer Master quickly. 
-QUEST_JOBSTEP_20150317_000083	Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help right? 
-QUEST_JOBSTEP_20150317_000084	There is one in the Miner's Village but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
+QUEST_JOBSTEP_20150317_000083	$Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help, right?
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:83]: I'd insert a "me" in there: "You're going to help me, right?"

File: QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
...
 QUEST_JOBSTEP_20150317_000079	This pouch...Did you come from the Hunter Master? {nl}Here. Give her this book. 
 QUEST_JOBSTEP_20150317_000080	I don't want to act mean when she's acting humble. 
 QUEST_JOBSTEP_20150317_000081	This leather will be enough. {nl}Go to the Quarrel Shooter Master before someone else takes it. 
 QUEST_JOBSTEP_20150317_000082	This rock will be useful in repairing the wall. {nl}Bring it to the Archer Master quickly. 
-QUEST_JOBSTEP_20150317_000083	Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help right? 
-QUEST_JOBSTEP_20150317_000084	There is one in the Miner's Village but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
+QUEST_JOBSTEP_20150317_000083	$Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help, right? 
+QUEST_JOBSTEP_20150317_000084	$There is one in the Miners' Village, but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv]: Tense? If it's not there anymore, then it _was_ there, not _is_ there.
"There was one in the Miners' Village, but it was lost when the Vubbes raided."

Line 68 uses "used to be", consider using the same wording in both (whichever you take)

File: QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
...
 QUEST_JOBSTEP_20150317_000087	The problem is, I'm broke. 
 QUEST_JOBSTEP_20150317_000088	You mean Lydia Schaffen's book that I have? {nl}I'll hand it over for 5000 silver.
 QUEST_JOBSTEP_20150317_000089	Well then, thanks for your help. {nl}Around 5000 silver will do. {nl}
 QUEST_JOBSTEP_20150317_000090	5000 silver...Yup, that's right. {nl}I'll give you the book as promised. 
-QUEST_JOBSTEP_20150317_000091	Well...the Archer Master must have the next book. {nl}He's still helping repair the wall so wouldn't he appreciate it if you got him some stones? {nl}I could take it to him myself but...owwww...my back.
-QUEST_JOBSTEP_20150317_000092	The Archer Master has the next book. {nl}He's not a mean person so he will give you the book if you ask nicely. {nl}
-QUEST_JOBSTEP_20150317_000093	I heard he is helping repair the damaged city wall. {nl}There are rocks delivered to the Miner's Village so he'll be moved if you bring him that rock. 
+QUEST_JOBSTEP_20150317_000091	$Well...the Archer Master must have the next book. {nl}He's still helping repair the wall, so wouldn't he appreciate it if you got him some rocks? {nl}I could take them to him myself, but...owwww...my back.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:91]: Missing space after ellipse. I'm seeing other lines like this, I guess it needs another pass for that.
Also: "He's still helping _to_ repair the wall." And the rock vs boulder thing.

File: QUEST_JOBSTEP.tsv [hunk 12]
```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
...
+QUEST_JOBSTEP_20150317_000092	$The Archer Master has the next book. {nl}He's not a mean person, so he will give you the book if you ask nicely. {nl}
+QUEST_JOBSTEP_20150317_000093	$I heard he is helping repair the damaged city wall. {nl}Rocks were delivered to the Miners' Village, and he'll be moved if you bring him one.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:93]: "he is helping _to_ repair the damaged city wall."

File: QUEST_JOBSTEP.tsv [hunk 13]
```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
...
-QUEST_JOBSTEP_20150317_000112	Isn't the book ready yet? {nl}I want to see all of Lydia Schaffen's book ASAP...
-QUEST_JOBSTEP_20150317_000113	Finally it's all here. {nl}I'll teach you my skills as promised. 
-QUEST_JOBSTEP_20150317_000114	I have some books to collect. It's Lydia Schaffen's books. {nl}I'll teach you my skills if you collect the books. What do you say? 
-QUEST_JOBSTEP_20150317_000115	The first book was in the Miner's Village but it disappeared after the Vubbes attacked. {nl}I assume the book is in the Vubbe Outpost? 
+QUEST_JOBSTEP_20150317_000112	$Isn't the book ready yet? {nl}I want to see all of Lydia Schaffen's book as soon as possible...
+QUEST_JOBSTEP_20150317_000113	$Finally, it's all here. {nl}I'll teach you my skills as promised. 
+QUEST_JOBSTEP_20150317_000114	$I have some books to collect. They're Lydia Schaffen's books. {nl}I'll teach you my skills if you collect the books. What do you say? 
+QUEST_JOBSTEP_20150317_000115	$The first book was in the Miners' Village, but it disappeared after the Vubbes attacked. {nl}I assume the book is in the Vubbe Outpost?
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:115]: Here, correct tense is used. I'm not sure the last '?' fits the context. They assume it's there, just like in other lines, so why is it a question? It would make sense if you could confirm it at that point, but otherwise?

File: QUEST_JOBSTEP.tsv [hunk 14]
```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
...
 QUEST_JOBSTEP_20150317_000121	Didn't the Ranger Master tell you anything?{nl}I asked him for Merog leather.
-QUEST_JOBSTEP_20150317_000122	Merogs are at the Fallen Worry Forest and the Quarrel Shooter Master is at Klaipeda. {nl}Someone may go there and get the book faster than you so please hurry.
+QUEST_JOBSTEP_20150317_000122	$Merogs are at the Fallen Worry Forest and the Quarrel Shooter Master is at Klaipeda. {nl}Someone may go there and get the book faster than you, so please hurry.
 QUEST_JOBSTEP_20150317_000123	The book is in good condition.{nl}Now, bring the book to the Archer Master. {nl}
 QUEST_JOBSTEP_20150317_000124	The Archer Master in Klaipeda has the last book. {nl}I heard he is repairing the wall, so this might help you get the book. 
-QUEST_JOBSTEP_20150317_000125	I heard there is a rock that will be useful for repairing the wall in the Miner's Village. {nl}Bring that to the Archer Master as a present. 
+QUEST_JOBSTEP_20150317_000125	$I heard there is a rock that will be useful for repairing the wall in the Miners' Village. {nl}Bring that to the Archer Master as a present.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:125]: rock/boulder

File: QUEST_JOBSTEP.tsv [hunk 15]
```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
...
+QUEST_JOBSTEP_20150317_000140	$We Peltastas promised the Highlanders we would lend our forces to rebuild the kingdom.{nl}So if you want to learn from me, you should also be responsible.{nl}
 QUEST_JOBSTEP_20150317_000141	Go meet Uska and lend your force.
 QUEST_JOBSTEP_20150317_000142	So you came here at the request of the Peltasta Master.{nl}The Highland Master once guaranteed that he would get support from the other masters. {nl}
 QUEST_JOBSTEP_20150317_000143	We, Peltastas, consider our duties as humans to be most important. 
 QUEST_JOBSTEP_20150317_000144	We, Peltastas, consider our duties as humans to be most important. 
 QUEST_JOBSTEP_20150317_000145	I was about to ask for aid in the search. {nl}
 QUEST_JOBSTEP_20150317_000146	I sent 4 scouts to the Spine Heart Dale, but they all disappeared.{nl}Maybe they didn't think the thorny forest was that dangerous... Please find them.
-QUEST_JOBSTEP_20150317_000147	Some issuse may have emerged.{nl}Please go to the Spine Heart Dale quickly.
+QUEST_JOBSTEP_20150317_000147	$Some issue may have emerged.{nl}Please go to the Spine Heart Dale quickly.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:147]: Do issues emerge or arise? I like the latter more...

File: QUEST_JOBSTEP.tsv [hunk 16]
```diff
@@ -152,29 +152,29 @@ QUEST_JOBSTEP_20150317_000151	I need to gather the energy of fire for my researc
...
-QUEST_JOBSTEP_20150317_000155	Did you say you'll find the book? {nl}Sorry but you're late. 
+QUEST_JOBSTEP_20150317_000155	$Did you say you'll find the book? {nl}Sorry, but you're late. 
 QUEST_JOBSTEP_20150317_000156	I heard rumors that it's in the Miners' Village, but I don't know the exact location of it. {nl}Maybe the Mayor would know...
 QUEST_JOBSTEP_20150317_000157	The Vubbes took all the books when they attacked. {nl}Maybe they will use it for fire in their outpost. 
 QUEST_JOBSTEP_20150317_000158	If there was something like that, we would have taken care of it. {nl}You go and try looking for it. 
-QUEST_JOBSTEP_20150317_000159	You found it. Great job. {nl}You'll have to work really hard to follow my trainings. 
-QUEST_JOBSTEP_20150317_000160	I heard the book was last seen in the Miners' Village. {nl}It's a very important book so maybe the Mayor might know.
+QUEST_JOBSTEP_20150317_000159	$You found it. Great job. {nl}You'll have to work really hard to follow my training. 
+QUEST_JOBSTEP_20150317_000160	$I heard the book was last seen in the Miners' Village. {nl}It's a very important book, so maybe the Mayor might know.
```
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv]: "so the Mayor might know." sounds unfinished.
"so the Mayor might know something about it."?

File: QUEST_JOBSTEP.tsv (hunk 6 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:16]: No, it's correct as is.  You could say "a research project", but saying "I'm doing research" is correct.

File: QUEST_JOBSTEP.tsv (hunk 11 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:91]: "helping repair" and "helping to repair" are equivalent, and both correct.

File: QUEST_JOBSTEP.tsv (hunk 13 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:115]: The '?' implies tone, that the person is unsure that their assumption is solid.  It makes sense, as the Vubbes stole it, but they're not totally sure where it ended up.  It's likely that it's in their outpost.

File: QUEST_JOBSTEP.tsv (hunk 11 above)
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:91]: Having a "to" there feels more complete, maybe it's not grammatically incorrect to leave it out, but IMHO, it flows better with the "to" present.

File: QUEST_JOBSTEP.tsv (hunk 13 above)
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:115]: Yes, but you usually do that if you expect the person to offer confirmation immediately. Plus, the two other instances of this don't use a question mark.

Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:115]: Aren't those other lines spoken by different people?

File: QUEST_JOBSTEP.tsv (hunk 8 above)
Comment (Azhurel) [inline QUEST_JOBSTEP.tsv:75]: I'm not sure if the name of the quest item is "rock," "boulder" or "stone." I also don't know if it's supposed to be plural. I switched everything to "rock" because that was the most common, even though it ~does~ sound kind of odd (bringing one rock at a time is silly!) It can easily be changed in the future if that's not what it's supposed to be.

File: QUEST_JOBSTEP.tsv (hunk 13 above)
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:115]: So far it's been one line per NPC unless it's a story, but in that case they use [] to denote different people's dialogue, no? I think it's a class Master speaking.

File: QUEST_JOBSTEP.tsv (hunk 7 above)
Comment (Azhurel) [inline QUEST_JOBSTEP.tsv]: "But there is a small load left on my mind...and I will resolve it now." also makes sense. As for the ellipsis, you don't have to put a space after it. Both are correct. However, I'll put a space after it. We need someone to go over all of them for the sake of consistency.

File: QUEST_JOBSTEP.tsv (hunk 10 above)
Comment (Azhurel) [inline QUEST_JOBSTEP.tsv]: Thanks, I totally missed that.

File: QUEST_JOBSTEP.tsv (hunk 15 above)
Comment (Azhurel) [inline QUEST_JOBSTEP.tsv:147]: They can emerge or arise; they're basically the same thing. In this case, it would be "Some issue may have arisen."

File: QUEST_JOBSTEP.tsv (hunk 16 above)
Comment (Azhurel) [inline QUEST_JOBSTEP.tsv]: I agree. Switched it.

File: QUEST_JOBSTEP.tsv (hunk 15 above)
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:147]: I have the feeling "arise" is more commonly used, but I might be mistaken.

File: QUEST_JOBSTEP.tsv (hunk 8 above)
Comment (Soukyuu) [inline QUEST_JOBSTEP.tsv:75]: If Korean is similar to Japanese, then there is no way to tell if something is singular or plural unless they explicitly specify it. I guess we can leave it as "rock" for now.

**Comment by imcgames** [comment]
all good for merge?

**Comment by Azhurel** [comment]
All good!

**Comment by HSGHHH** [comment]
This looks good to me.

**Comment by terrapin47** [comment]
@Azhurel this branch has conflicts and can't be merged.

**Comment by Azhurel** [comment]
Is there an easy way to resolve the conflicts?

**Comment by terrapin47** [comment]
Here's a guide:
https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/

**Comment by terrapin47** [comment]
Oh, you'll want to make sure your master branch is up to date, and then update your patch-1 branch off of that.  Then you'll see the conflicts, and be able to fix them.

**Comment by terrapin47** [comment]
Or just accept my pull requests onto your fork's branches.

**Comment by Azhurel** [comment]
Thank you! I was getting a little dizzy trying to figure out how to do it myself.

**Comment by imcgames** [comment]
Thank you :smile:
